lib/ldap_fluff/generic_member_service.rb

```diff
@@ -70,7 +70,7 @@ def get_logins(userlist)
 
   def get_login_from_entry(entry)
     [@attr_login, 'uid', 'cn'].each do |attribute|
-      return entry.send(attribute) if entry.respond_to? attribute
+      return entry.send(attribute.map(&:downcase)) if entry.respond_to? attribute
```

Comment:
I'd guess you'd actually want to do `entry.send(attribute).map(&:downcase)`.

Though why exactly are you adding this last commit?  The rest makes sense, not sure I understand what this fixes though.